src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/messaging/info/group/GroupDumpResponse.java

```diff
@@ -26,21 +29,38 @@
 import lombok.Value;
 
 import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
 @Value
 @EqualsAndHashCode(callSuper = false)
-public class GroupDumpResponse extends InfoData {
-    @JsonProperty("switch_id")
-    SwitchId switchId;
+public class GroupDumpResponse extends InfoData implements Chunkable<GroupDumpResponse> {
 
     @JsonProperty("group_speaker_data")
     List<GroupSpeakerData> groupSpeakerData;
 
     @JsonCreator
     @Builder
-    public GroupDumpResponse(@JsonProperty("switch_id") SwitchId switchId,
-                             @JsonProperty("group_speaker_data") List<GroupSpeakerData> groupSpeakerData) {
-        this.switchId = switchId;
+    public GroupDumpResponse(@JsonProperty("group_speaker_data") List<GroupSpeakerData> groupSpeakerData) {
         this.groupSpeakerData = groupSpeakerData;
     }
+
+    @Override
+    public List<GroupDumpResponse> split(int chunkSize) {
```

Comment:
Can this be moved to Chunkable as default method?

Reply:
No, because each object is unique. one contains a list inside and the other one contains 4 other objects. Each of them need to have separate split method. 

Moreover to make it default Chunkable must have access to inner field `groupSpeakerData`. But it has no such access because it specific filed for only this class. other classes have other fields